mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1390,7 +1390,7 @@ def predict_with_latency(X_copy):
             elif isinstance(sample_pred, list):
                 return sum(y_pred_list, [])
             elif isinstance(sample_pred, pd.Series):
-                return pd.concat(y_pred_list)
+                return pd.concat(y_pred_list, ignore_index=True)
```

Comment:
Do we need pd.concat here since we are not concating anything as far as I can tell. 

Maybe we can switch this to pd.DataFrame so convert list to Data frame?

Let me know if I am missing something

Reply:
Sorry, not sure what you mean by we are not concating anything?
I think it is better to return a pd.Series given that the model's predict() function returns a pd.Series, so that we have the same model_predictions regardless of if the latency is calculated or not.

Reply:
```
if compute_latency:
    model_predictions = predict_with_latency(X_copy)
else:
    model_predictions = self.model.predict(X_copy)
```

Ideally model_predictions should be the same in these two cases, so that computing the latency doesn't affect what the predictions looks like. LMK if there are any issues here - will first merge this PR and can follow up if there are concerns